shared/src/bundles/videos/types/create-video-request-dto.type.ts

```diff
@@ -1,7 +1,9 @@
 type CreateVideoRequestDto = {
     userId: string;
     name: string;
-    url: string;
+    url?: string;
+    composition: string;
+    previewUrl: string;
```

Comment:
I am not sure if we need to send `previewUrl` when creating video. I guess we should define this property based on first scene of the composition that frontend sends (could be done in separate ticket).

Reply:
Also url property will be set only on backend after video rendering and I am not sure we need it in CreateVideoRequestDto. Or should we leave it for testing purposes?

Reply:
Agree, we don't have url yet when creating video

Reply:
I've already solved it in my next PR https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/318